src/praisonai-agents/praisonaiagents/goal/judge.py

```
@@ -0,0 +1,152 @@
+    prompt = _build_goal_judge_prompt(state, transcript_tail)
+    try:
+        judge = Judge(model=judge_model or _default_judge_model(), temperature=0.0)
+        litellm = judge._get_litellm()
+        response = litellm.completion(
+            model=judge.model,
+            messages=[{"role": "user", "content": prompt}],
+            temperature=0.0,
+            max_tokens=300,
+        )
+        raw = response.choices[0].message.content or ""
+    except Exception as exc:  # pragma: no cover - network/optional dep
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)